sdk/main/include/Hbar.h

```diff
@@ -32,6 +33,10 @@ namespace Hedera
  * Implemented as a wrapper class to force handling of units. Direct interfacing with Hedera accepts amounts in tinybars
  * however the nominal unit is hbar.
  */
+
+// Define the pattern
+const std::regex FROM_STRING_PATTERN("^((?:\\+|\\-)?\\d+(?:\\.\\d+)?)(\\ (tℏ|μℏ|mℏ|ℏ|kℏ|Mℏ|Gℏ))?$");
```

Comment:
I don't think this needs to be exposed in the header file, you could just throw it in the source file within an anonymous namespace

Reply:
Fixed in af94ce7e0647ed4502fa8593960ba403160dea5d